src/views/FileBrowserView/PublishFileBrowser.vue

```diff
@@ -173,6 +218,29 @@ const sortByName = (a, b) => {
   return 0;
 };
 
+const EXTERNAL_SERVICES = [
+  {
+    name: 'Bioimagesuite/Viewer',
+    regex: '.nii(.gz)?$',
+    extensions: [
+      '.nii',
+      '.nii.gz',
+    ],
+    maxsize: 4e6,
```

Comment:
Should this be 4e9 (for 4 GB)? 4 MB seems pretty restrictive considering the typical file sizes on dandi

Reply:
you're right it is too restrictive...